### ISSUE TITLE
Examples Don't work

### ISSUE THREAD

**Issue by AdvanceXplorer** [HumanCompatibleAI/imitation#815]
Hi,

I am using Python 3.10.9 with gym = 0.21.0. When i try to run imitation-master\examples\quickstart.py, I get the following error:

Traceback (most recent call last):
File "C:\Users\advan\anaconda3\lib\site-packages\gym\envs\registration.py", line 158, in spec
return self.env_specs[id]
KeyError: 'seals/CartPole-v0'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "c:\Users\advan\OneDrive\Masaüstü\imitation-master\examples\quickstart.py", line 17, in
env = make_vec_env(
File "C:\Users\advan\anaconda3\lib\site-packages\imitation\util\util.py", line 107, in make_vec_env
spec = gym.spec(env_name)
File "C:\Users\advan\anaconda3\lib\site-packages\gym\envs\registration.py", line 239, in spec
return registry.spec(id)
File "C:\Users\advan\anaconda3\lib\site-packages\gym\envs\registration.py", line 185, in spec
raise error.DeprecatedEnv(
gym.error.DeprecatedEnv: Env seals/CartPole-v0 not found (valid versions include ['CartPole-v0', 'CartPole-v1']),

which is the same as:
https://imitation.readthedocs.io/_/downloads/en/stable/pdf/).readthedocs.io/_/downloads/en/stable/pdf/.

How can i fix this?

**Comment by nil123532** [comment]
same

**Comment by Rajesh-Siraskar** [comment]
I too am facing issue running [Train_BC notebook](https://github.com/HumanCompatibleAI/imitation/blob/master/docs/tutorials/1_train_bc.ipynb)

This occurs when running code in the first cell:
```
env = make_vec_env(
    "seals:seals/CartPole-v0",
    rng=np.random.default_rng(),
    post_wrappers=[
        lambda env, _: RolloutInfoWrapper(env)
    ],  # needed for computing rollouts later
)
```

**Error output**
```
---------------------------------------------------------------------------
NamespaceNotFound                         Traceback (most recent call last)
Cell In[1], line 7
      4 from imitation.util.util import make_vec_env
      5 from imitation.data.wrappers import RolloutInfoWrapper
----> 7 env = make_vec_env(
      8     "seals:seals/CartPole-v0",
      9     rng=np.random.default_rng(),
     10     post_wrappers=[
     11         lambda env, _: RolloutInfoWrapper(env)
     12     ],  # needed for computing rollouts later
     13 )
     14 expert = load_policy(
     15     "ppo-huggingface",
     16     organization="HumanCompatibleAI",
     17     env_name="seals/CartPole-v0",
     18     venv=env,
     19 )

File c:\users\rajeshs\appdata\local\programs\python\python39\lib\site-packages\imitation\util\util.py:107, in make_vec_env(env_name, rng, n_envs, parallel, log_dir, max_episode_steps, post_wrappers, env_make_kwargs)
     82 """Makes a vectorized environment.
     83 
     84 Args:
   (...)
    103     A VecEnv initialized with `n_envs` environments.
    104 """
    105 # Resolve the spec outside of the subprocess first, so that it is available to
    106 # subprocesses running `make_env` via automatic pickling.
--> 107 spec = gym.spec(env_name)
    108 env_make_kwargs = env_make_kwargs or {}
    110 def make_env(i: int, this_seed: int) -> gym.Env:
    111     # Previously, we directly called `gym.make(env_name)`, but running
    112     # `imitation.scripts.train_adversarial` within `imitation.scripts.parallel`
   (...)
    117     # work. For more discussion and hypotheses on this issue see PR #160:
    118     # https://github.com/HumanCompatibleAI/imitation/pull/160.

File c:\users\rajeshs\appdata\local\programs\python\python39\lib\site-packages\gym\envs\registration.py:699, in spec(env_id)
    697 if spec_ is None:
    698     ns, name, version = parse_env_id(env_id)
--> 699     _check_version_exists(ns, name, version)
    700     raise error.Error(f"No registered env with id: {env_id}")
    701 else:

File c:\users\rajeshs\appdata\local\programs\python\python39\lib\site-packages\gym\envs\registration.py:219, in _check_version_exists(ns, name, version)
    216 if get_env_id(ns, name, version) in registry:
    217     return
--> 219 _check_name_exists(ns, name)
    220 if version is None:
    221     return

File c:\users\rajeshs\appdata\local\programs\python\python39\lib\site-packages\gym\envs\registration.py:187, in _check_name_exists(ns, name)
    185 def _check_name_exists(ns: Optional[str], name: str):
    186     """Check if an env exists in a namespace. If it doesn't, print a helpful error message."""
--> 187     _check_namespace_exists(ns)
    188     names = {spec_.name for spec_ in registry.values() if spec_.namespace == ns}
    190     if name in names:

File c:\users\rajeshs\appdata\local\programs\python\python39\lib\site-packages\gym\envs\registration.py:182, in _check_namespace_exists(ns)
    173 suggestion = (
    174     difflib.get_close_matches(ns, namespaces, n=1) if len(namespaces) > 0 else None
    175 )
    176 suggestion_msg = (
    177     f"Did you mean: `{suggestion[0]}`?"
    178     if suggestion
    179     else f"Have you installed the proper package for {ns}?"
    180 )
--> 182 raise error.NamespaceNotFound(f"Namespace {ns} not found. {suggestion_msg}")

NamespaceNotFound: Namespace seals:seals not found. Have you installed the proper package for seals:seals?

```

**Comment by yunlong-wang-cn** [comment]
same

**Comment by Charles-Lim93** [comment]
Same here. It seems have problem with Seals..

**Comment by ernestum** [comment]
Hi, thanks for reaching out. We don't support `gym` any more. Try running with `gymnasium`. Also make sure you have at least `seals` version 0.2.1 installed.

Note also that I found a (unrelated) bug in the `quickstart.py`, that is addressed in #823. So until this is merged I recommend to try that branch.
